examples/with-sitemap/public/sitemap.xml

```diff
@@ -0,0 +1,13 @@
+
+  <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+            
+                        <url>
+                            <loc>http://localhost:3000/contact</loc>
+                        </url>
+                    
+                        <url>
+                            <loc>http://localhost:3000</loc>
+                        </url>
+                    
+              </urlset>
+    
```

Comment:
Update this file after accepting the changes to `generate-sitemap.js`, it should be this:

```suggestion
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
  <url>
    <loc>http://localhost:3000/contact</loc>
    <changefreq>hourly</changefreq>
  </url>
  <url>
    <loc>http://localhost:3000</loc>
    <changefreq>hourly</changefreq>
  </url>
</urlset>
```

Reply:
@lfades Since we are using `.env` for Website Url, isn't it better to also change the links here from being hardcoded to `process.env.WEBSTE_URL`?